Octokit.Reactive/Clients/IObservableUsersClient.cs

```diff
@@ -28,6 +28,13 @@ public interface IObservableUsersClient
         /// <returns>A <see cref="User"/></returns>
         IObservable<User> Update(UserUpdate user);
 
+        /// <summary>
+        /// Returns list of all <see cref="User"/>, in the order that they signed up on GitHub after specified ID.
+        /// </summary>
+        /// <param name="since">ID after which the returned list begin</param>
+        /// <returns>A list of <see cref="User"/> after specified ID</returns>
+        IObservable<IReadOnlyPagedCollection<User>> GetAll(string since);
```

Comment:
This is not really important for the discussion but we generally "flatten" observables to not return lists, so I pushed a commit to change this to return `IObservable<User>` as that will be more familiar to callers of the API.